pkg/commands/run.go

```diff
@@ -313,8 +313,24 @@ func fixSlicesFlags(fs *pflag.FlagSet) {
 			return
 		}
 
+		// custom join to handle string with comma.
+		var g string
+		for i, v := range s {
+			if strings.Contains(v, ",") {
+				// add quotes to escape comma because spf13/pflag use a CSV parser:
+				// https://github.com/spf13/pflag/blob/85dd5c8bc61cfa382fecd072378089d4e856579d/string_slice.go#L43
+				g += `"` + v + `"`
+			} else {
+				g += v
+			}
+
+			if i < len(s)-1 {
+				g += ","
+			}
+		}
+
 		// calling Set sets Changed to true: next Set calls will append, not overwrite
-		_ = f.Value.Set(strings.Join(s, ","))
+		_ = f.Value.Set(g)
```

Comment:
Wouldn't it be easier to read and less complex if we just added quotes around all flags so the CSV parser would just do the right thing? 

```suggestion
		// add quotes to escape comma because spf13/pflag use a CSV parser:
		// https://github.com/spf13/pflag/blob/85dd5c8bc61cfa382fecd072378089d4e856579d/string_slice.go#L43
		for i, v := range s {
			s[i] = "` + v + `"`
		}
		

		_ = f.Value.Set(strings.Join(s, ","))
```

Or if we need to keep `s`, create a new `[]string` and insert the quoted flags to those.

Reply:
It's less efficient and less precise but it's possible.

Reply:
Doesn't `strings.Contains` have to iterate over every byte in every flag just do determine the branch? Also flags are limited in amount so I can't imagine one could save more than a couple of microseconds on this difference and thus for me personally readability and maintainability is more important.

Reply:
I don't know why but the suggestion buttons are disabled so I cannot accept your suggestion, so I will do it by hand.